.github/workflows/sonargate.yml

```diff
@@ -0,0 +1,44 @@
+name: sonargate
+on: [pull_request]
+jobs:
+  sonargate:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Java11
+      uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+    - name: Install confd
+      run: |
+        sudo apt-get install wget
+        wget https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-amd64
+        mkdir -p /opt/confd/bin
+        mv confd-0.16.0-linux-amd64 /opt/confd/bin/confd
+        chmod +x /opt/confd/bin/confd
+    - name: Gradle cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+        restore-keys: ${{ runner.os }}-gradle-
+    - name: Prepare gradle props
+      run: |
+        mkdir -p $HOME/.gradle
+        echo "org.gradle.jvmargs=-Xmx4096M" >> $HOME/.gradle/gradle.properties
+        echo "org.gradle.console=plain" >> $HOME/.gradle/gradle.properties
+        echo "systemProp.sonar.host.url=https://sonarcloud.io" >> $HOME/.gradle/gradle.properties
+        echo "systemProp.sonar.organization=open-kilda" >> $HOME/.gradle/gradle.properties
+        echo "systemProp.sonar.projectKey=org.openkilda:kilda-parent" >> $HOME/.gradle/gradle.properties
+        echo "systemProp.sonar.login=${{secrets.SONAR_TOKEN}}" >> $HOME/.gradle/gradle.properties
+        echo "systemProp.sonar.password=" >> $HOME/.gradle/gradle.properties
+        cat $HOME/.gradle/gradle.properties
+        rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+        rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+    - name: Run sonar
+      run: |
+        export TRAVIS=true
```

Comment:
we removed `TRAVIS` var from scripts. So it is unnecessary now

Reply:
thx, I will test this out.